### DISCUSSION TITLE
Can I use a different page besides bookmarks?

### DISCUSSION THREAD

**Discussion by kadeerae** [nianeyna/ao3downloader#176] [Q&A]
The downloader is currently running so I can't check it, but if I give it the URL to my subscribed works would it download the works from that as well? I didn't really start bookmarking until recently so I'm trying to save my subscribed works as well

**Comment by nianeyna** [comment]
yes. you can use almost any ao3 url. the only exceptions are urls that don't support multiple pages of results - for example: the ao3 home page and user dashboard pages (user *works* pages are fine however)

note that when downloading your subscriptions specifically, it can get subscribed works and series but not subscribed authors, so if you are subscribed to any authors you will need to back those up separately.
